packages/website/src/components/Navbar/Navbar.tsx

```diff
@@ -8,13 +9,22 @@ import { useRecruitmentModal } from '@/components/RecruitmentModal';
 
 import { DesktopBaseNavbar, MobileBaseNavbar } from './BaseNavbar';
 
+const headroomStyles = {
+  zIndex: 1000,
+  background: 'rgba(255, 255, 255)',
+  boxShadow: '1px 1px 1px rgba(0,0,0,0.25)',
```

Comment:
❓ Do we need box shadow here? I've noticed that in the mobile view, navigating behaves a bit strange.

https://user-images.githubusercontent.com/38866668/131527225-2ced7716-d51b-48ee-b843-cf2a0123b478.mov